integrationtest/mapper/orm-coordination-outbox-polling/src/test/java/org/hibernate/search/integrationtest/mapper/orm/coordination/outboxpolling/automaticindexing/OutboxPollingAutomaticIndexingConcurrencyIT.java

```diff
@@ -70,7 +71,7 @@ public void setup() {
 
 	private SessionFactory setup(String hbm2ddlAction) {
 		backendMock.expectSchema( IndexedEntity.NAME, b -> b
-				.field( "text", String.class, f -> f.analyzerName( AnalyzerNames.DEFAULT ) ) );
+				.field( "text", String.class, f -> f.analyzerName( AnalyzerNames.DEFAULT ).highlightable( Highlightable.DEFAULT ) ) );
```

Comment:
Okay, having to do this everywhere will get old real fast.

Can keep your changes to the backend, and restore the code you had on the mapper side that would *not* forward `highlightable` when it's just `DEFAULT`?

That way we won't have to do this everywhere, and you can revert these changes.

Reply:
Hah 😄 🙈 now I wish I'd made these changes in their own commit 😆 but no worries will change things back 😃